torchreid/integration/sc/parameters.py

```diff
@@ -56,6 +56,32 @@ class __LearningParameters(ParameterGroup):
             affects_outcome_of=ModelLifecycle.TRAINING
         )
 
+    @attrs
+    class __InferenceParameters(ParameterGroup):
+        header = string_attribute("Parameters for inference")
+        description = header
+
+        class_name = selectable(default_value=Models.CLASSIFICATION,
+                                header="Model class for inference",
+                                description="Model classes with defined pre- and postprocessing",
+                                editable=False,
+                                visible_in_ui=True)
+
+        @attrs
+        class __Postprocessing(ParameterGroup):
+            header = string_attribute("Postprocessing")
+            description = header
+
+            topk = configurable_integer(
+                header="Top k elements",
```

Comment:
That's not the SC style of visualization/not applicable for multilabel and hierarchical cases. Perhaps one could try web UI on a working SC instance to get some inspiration if the the exportable code aims to mimic it somehow.